src/index.js

```diff
@@ -0,0 +1,26 @@
+import 'react-hot-loader/patch';
+import React from 'react';
+import { render } from 'react-dom';
+import { AppContainer } from 'react-hot-loader';
+import App from './app.jsx';
+
+const renderNew = Component => {
+	render(
+		<AppContainer>
+			<Component />
+		</AppContainer>,
+		document.getElementById('main')
+	);
+}
+
+renderNew(App);
+
+if (module.hot) {
+	module.hot.accept('./app.jsx', () => {
+		const NextApp = require('./app.jsx').default;
```

Comment:
Can this be refactored into a one liner by removing `NextApp`?

Reply:
if possible why not, but I don't think we can remove NextApp or maybe you have some idea?